MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.ListBox.xaml

```diff
@@ -159,10 +159,10 @@
   </Style>
 
   <Style x:Key="MaterialDesignToolToggleListBox" TargetType="{x:Type ListBox}">
-    <Setter Property="Background" Value="{DynamicResource MaterialDesignToolBarBackground}" />
+    <Setter Property="Background" Value="{DynamicResource MaterialDesign.Brush.ToolBar.Background}" />
```

Comment:
Should this be target type specific?

```suggestion
    <Setter Property="Background" Value="{DynamicResource MaterialDesign.Brush.ListBox.Background}" />
```

Reply:
In this case I think the ToolBar.Background makes more sense as this style is expected to be used inside of a ToolBar.